pgo-rmdata/rmdata/process.go

```diff
@@ -254,7 +256,7 @@ func removeClusterConfigmaps(request Request) {
 	// log the fact there was an error; this function is just a pass through
 	for _, cm := range clusterConfigmaps {
 		if err := request.Clientset.CoreV1().ConfigMaps(request.Namespace).Delete(cm, &metav1.DeleteOptions{}); err != nil {
-			log.Error(err)
+			log.Warn(err)
```

Comment:
Wondering if it would make sense to add a `kerrors.IsNotFound` check here to simply not log anything in the event that the resource couldn't be found (i.e. for when pgBouncer isn't enabled).  This way we can still log a proper error when an error occurs when attempting to delete those ConfigMaps that should always be present.

Reply:
I still think it's only a warning even if it's a "proper error" -- we're still continuing on with the rmdata process even if something else happens.

Happy to include the check to reduce the number of warnings.

Reply:
Adding in that check and leaving it as Error.